app/assets/javascripts/admin/views/user/team_role_modal_view.js

```diff
@@ -168,7 +180,7 @@ class TeamRoleModalView extends React.PureComponent {
         <Col span={12}>
           {this.getRoleComponent(team)}
         </Col>
-      </Row>,
+      </Row>
```

Comment:
Are you using the sublime package JsPrettier? I think you have to configure it like this:

```
	"prettier_options": {
		"printWidth": 100,
		"trailingComma": "all",
	}
```

since these are the settings used by `yarn pretty`. It's a shame that these options can't be saved centrally so that the ST package can just pick it up.